FFWEB-1803: Publish `dispatchRaw` function (#270)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/documentation/server-side-rendering.md

**Note**: You are not just limited to mustache. You can use any template engine your platform supports. The advantage of using mustache here is that you can use the same templates as in Web Components.  

#### Sending the response

> The response to the browser contains the pre-rendered HTML and raw response available to be used by Web Components.
> Most convenient way to make the response available to use by the Web Components is to store it as JSON in a variable.

Reviewer: ```suggestion
The response to the browser contains the pre-rendered HTML and the raw response ready to be used by Web Components.
The most convenient way to make the response available to use by Web Components is to store it as JSON in a variable.
```

```html

> <script>
> const rawResponse = JSON.parse(RESPONSE_TEXT);
> </script> 

Reviewer: Let's indent this to be consistent with other listings.
There is another listing that could be indented on line 50.
```suggestion
<script>
    const rawResponse = JSON.parse(RESPONSE_TEXT);
</script> 
```

```

> **Note**: `RESPONSE_TEXT` in example above represents response text received from FACT-Finder. Passing it to variable depends on the used framework.

Reviewer: ```suggestion
**Note**: `RESPONSE_TEXT` in the example above represents the response text received from FACT-Finder. The way it is passed to a variable depends on the used framework.
```

**Note**: You are not just limited to mustache. You can use any template engine your platform supports. The advantage of using mustache here is that you can use the same templates as in Web Components.  

#### Sending the response

> The response to the browser contains the pre-rendered HTML and raw response available to be used by Web Components.

Author: added more info

    <template data-role="record">
        <!--put `ff-record` element template here  --->
    </template>
```
Check the `ff-record-list` [documentation](/api/3.x/ff-record-list#tab=docs) for more details about its template.

> After doing this, the only thing left to do is to dispatch response to the Web Components using `dispatchRaw`.
> See the [Result Dispatcher](/api/3.x/core-result-dispatcher) for more details regarding this utility.
> Dispatching response via `dispatchRaw` will trigger `ff-record-list` to replace pre-rendered records with the actual ones. 

Reviewer: ```suggestion
After doing this, the only thing left to do is to dispatch the response to Web Components using `dispatchRaw`.
See [Result Dispatcher](/api/3.x/core-result-dispatcher) for more details regarding this utility.
Dispatching the response via `dispatchRaw` will trigger `ff-record-list` to replace pre-rendered records with the actual ones. 
```

>     <template data-role="record">
>         <!--put `ff-record` element template here  --->
>     </template>

Reviewer: While we are at it.
```suggestion
<template data-role="record">
    <!--put `ff-record` element template here  --->
</template>
```
